src/App.js

```diff
@@ -1,8 +1,22 @@
-import React from 'react'
+import React, { useState } from 'react'
 import Header from './Header'
+import QuestionsForm from './QuestionsForm'
+import Summary from './Summary'
 
 function App() {
-  return <Header />
+  const [answers, setAnswers] = useState([])
+
+  return (
+    <>
+      <Header />
+      <QuestionsForm onClick={addAnswers} />
+      <Summary answers={answers} />
+    </>
+  )
+
+  function addAnswers(newAnswers) {
```

Comment:
Do you need this function? It seems like it only calls the setAnswers function. In that case I think you could directly pass setAnswers as the prop.

Reply:
True - my thought on it was to improve readability by naming the function "addAnswers"